src/mixins/observable.mixin.js

```diff
@@ -46,6 +46,27 @@
     return this;
   }
 
+  function _once(eventName, handler) {
+    var _handler = function () {
+      handler();
+      this.off(eventName, _handler);
```

Comment:
I wonder how this work with hoisting.
We are referencing the function before we finish create it.
Is not a question about your code but more on javascript.

Reply:
I guess I can call the function directly and not from the mixin